sites/main-site/src/pages/index.astro

```diff
@@ -127,6 +133,11 @@ import AnnouncementBanner from '@components/navbar/AnnouncementBanner.astro';
         </div>
     </div>
 
+    <div class="container pb-5 text-center lead" id="features">
+        <p>nf-core pipelines are <strong>community owned</strong>.</p>
+        <p>Everyone is welcome to contribute, and maintainers can change over time.</p>
```

Comment:
```suggestion
        <p>Everyone is welcome to use the project and contribute to its growth.</p>
```

Reply:
Reads better, but changes the meaning a bit. I was trying to follow on from the "community owned" bit to emphasise that people can come and go..

Reply:
The word "maintainers" won't mean much to someone new to the community. Maybe something like "Everyone is welcome to use, contribute to, and help maintain this project over time"

Reply:
How about this?
```suggestion
        <p>Everyone is welcome to use, contribute to, and help maintain pipelines.</p>
```

Reply:
Maybe not just pipelines...? the project in general?

Reply:
Because people can also contribute to infra/docs etc.

Reply:
ok, will change pipelines -> code on the line above and use @christopher-hakkaart 's text with project on this line.

Reply:
Done in 854dc9ba